src/csharp/Microsoft.Spark/Utils/PythonSerDe.cs

```diff
@@ -18,14 +18,18 @@ namespace Microsoft.Spark.Utils
     /// </summary>
     internal class PythonSerDe
     {
+        private static readonly RowConstructor s_rowConstructor;
```

Comment:
can you please add a comment on thread safety here so that it's clear without looking at `RowConstructor` implementation?

Reply:
Added comment.

Reply:
don't you need to add `why` there is no issue when multiple threads accesing this one static object? :)